docs/resource_factories.md

```diff
@@ -245,7 +245,16 @@ use Sylius\Resource\Metadata\Create;
 #[Create(
     factory: [BookFactory::class, 'create'], 
 )]
-class Book implements \Sylius\Resource\Model\ResourceInterface
+use App\Factory\BookFactory;
```

Comment:
Double `use` from lines 240-247 to this snippet.

Reply:
done